frontend/src/queries/nodes/InsightQuery/utils/queryNodeToFilter.ts

```diff
@@ -256,15 +256,20 @@ export const queryNodeToFilter = (query: InsightQueryNode): Partial<FilterType>
         camelCasedRetentionProps.returning_entity = queryCopy.retentionFilter?.returningEntity
         camelCasedRetentionProps.target_entity = queryCopy.retentionFilter?.targetEntity
         camelCasedRetentionProps.total_intervals = queryCopy.retentionFilter?.totalIntervals
-        camelCasedRetentionProps.show_mean = queryCopy.retentionFilter?.showMean
+        camelCasedRetentionProps.show_mean =
```

Comment:
why do we need to add this in if we're removing the field? 😅 
UI doesn't rely on this field anymore I would think

Reply:
yeah, that's right. i just added it for completeness i guess. we should eventually remove the `queryNodeToFilter` function all-together (only used for converting new series format to old one for action filters now).